tools/pd-ctl/pdctl/command/config_command.go

```diff
@@ -232,10 +233,24 @@ func showClusterVersionCommandFunc(cmd *cobra.Command, args []string) {
 
 func postConfigDataWithPath(cmd *cobra.Command, key, value, path string) error {
 	var val interface{}
+	var err error
 	data := make(map[string]interface{})
-	val, err := strconv.ParseFloat(value, 64)
-	if err != nil {
-		val = value
+	if strings.Contains(value, "[") {
```

Comment:
if value is `[zone[1,2],zone[2,3]]`?

Reply:
It will print `invalid label` error.

> what happened if config set location-labels "" after the pr

It will print the help message.